UI/Web/src/app/admin/changelog/changelog.component.html

```diff
@@ -1,7 +1,10 @@
 <ng-container *ngFor="let update of updates; let indx = index;">
     <div class="card w-100 mb-2" style="width: 18rem;">
         <div class="card-body">
-          <h5 class="card-title">{{update.updateTitle}}&nbsp;<span class="badge badge-secondary" *ngIf="update.updateVersion === update.currentVersion">Installed</span></h5>
+          <h5 class="card-title">{{update.updateTitle}}&nbsp;
+              <span class="badge badge-secondary" *ngIf="update.updateVersion <= update.currentVersion">Installed</span>
```

Comment:
We can use an else statement on the ngIf to avoid potential bugs and duplicate logic.
```
*ngIf="update.updateVersion <= update.currentVersion; else available"
<ng-template #available>
 <span class="badge badge-secondary">Available</span>
          </h5>
</ng-template>
```

Reply:
Didn't realize if...else logic was available in Angular templating. That's pretty awesome and good to know. Made the change.